src/router/paths.js

```diff
@@ -40,7 +40,7 @@ export default [
     }
   },
   {
-    path: '/workflows/:workflowName',
+    path: '/workflows/:workflowName(.*)',
```

Comment:
Is there any danger of `.*` being too wide-open in this context?  Or is the default "anything except `/`" i.e. `[^/]*`, in which case `.*` is the only way to extend it :grin: )

Reply:
Good question. I tried some variations, but will take a better look tomorrow morning.

Vue Router uses [this library](https://github.com/pillarjs/path-to-regexp/tree/v1.7.0). I will take a look at its documentation and see what options we have.

Reply:
>Is there any danger of .* being too wide-open in this context?

It looks like that will indeed capture everything after `/workflows/` until the next `/`, from [their docs](https://github.com/pillarjs/path-to-regexp/tree/v1.7.0#named-parameters) "By default, the parameter will match until the following path segment."

The `(.*)` here captures everything until the end of the URL. It is smart enough to stop if it sees a query like `?param=value`, or a fragment like `#anotherfragment`.

So it is greedy, and captures the rest of the URL. Not sure if it has any potential risk of capturing more than we want.

But this regex works too: `/workflows/:workflowName([a-zA-Z0-9_/-])*`

Should we switch to this one? Not sure if I forgot to include some character, but we can just add more if necessary

Reply:
What I was wondering is, does the default already capture "any character except for `/`" - if it does, then `.*` really only adds one more character (i.e. `\`) and it seems like there would be no point in using the more specific expression (which would presumably exclude some characters that are not currently excluded)... If so, then I think `.*` is fine.  Does that make sense :grimacing: ?

Reply:
> > Is there any danger of .* being too wide-open in this context?
> It looks like that will indeed capture everything after /workflows/ until the next /, from their docs "By default, the parameter will match until the following path segment."

(Maybe you answered my question there already - but it looks like you're reply to my question about `.*` ... but then talking about "the default" - so I'm a bit confused!)

Reply:
Yup! All good. And it was good to dig a bit deeper and understand how `path-to-regexp` works. Handy library, might be useful later.

Reply:
So we can stick with `.*`?

Reply:
The default for named parameters. In Vue Router, we have `/:workflowName`. `path-to-regexp` will parse anything after the slash symbol, until it finds another slash or end of stream, and create the JS object `{ workflowName: "whatever-user-entered" }`.

Vue Router uses that to pass that value as a `prop` to the View.

The `.*` changes that so that it keeps consuming the stream until it is empty, even if it sees a slash. So `/a/b/c` will generate `{ workflowName: "a/b/c" }`.

Reply:
> So we can stick with `.*`?

I'm +1 for `.*`. (we are missing each other in the comments here haha, sorry)

Reply:
Great (and sorry for complicating this small change!)